ports/README.md

```diff
@@ -28,11 +28,8 @@ After copying the port, add an entry to the table below. You should also contrib
 
 ## List of port overlays
 
-| Port                       | Reason                                                                                                  |
-|----------------------------|---------------------------------------------------------------------------------------------------------|
-| `libmagic`                 | Updating to the upstream port deferred due to failures.                                                 |
-| `openssl`                  | Pinning to OpenSSL 1.1 until we can move to 3.0 in January 2024.                                        |
-| `pcre2`                    | To be removed alongside libmagic.                                                                       |
-| `azure-storage-common-cpp` | Patching to disable default features on libxml2 (https://github.com/Azure/azure-sdk-for-cpp/pull/5221). |
-| `libfaketime`              | Port does not yet exist upstream                                                                        |
-| `vcpkg-cmake-config`       | Patching to fix build issues with CMake 3.29.1. (https://github.com/microsoft/vcpkg/pull/38017)         |
```

Comment:
Subsequent versions of CMake 3.29.x have fixed the breaking change and this custom port is no longer needed.